### ISSUE TITLE
Changing test directory location

### ISSUE THREAD

**Issue by atztogo** [atztogo/phonopy#77]
@jochym. I am considering to move ``phonopy/test/phonopy/*`` to ``phonopy/test/*``. Is it OK for you? Does this affect to your conda-forge setting very much?

**Comment by jochym** [comment]
Yes - it does but it is minimal - just do it and I'll fix the problems.

**Comment by atztogo** [comment]
I did this.
